dz_dynamodb/sync_strategies/log_based.py

```diff
@@ -147,6 +156,13 @@ def sync(config, state, stream):
     table_name = stream['tap_stream_id']
 
     client = dynamodb.get_client(config)
+    # for open shards their will be a continous loop while getting records so to handle that max_iteration_for_open_shards is used
+    found_max_iteration_for_open_shards = False
+    max_iteration_for_open_shards = 1000
+    if config.get('max_iteration_for_open_shards') :
+         # override max_iteration_for_open_shards with config value
+         max_iteration_for_open_shards = config.get('max_iteration_for_open_shards')
+         found_max_iteration_for_open_shards = True
```

Comment:
lets enable it by default

Reply:
for testing purpose now so that the client running now not have to face any issue due to it